### PR TITLE
Add more programmer descriptions

### PR DESCRIPTION
**Pull request by MCUdude**
#1437 related.

Please proof read and review. We don't have to merge this before the 7.3 release, but I don't see how this could break anything.

### CODE REVIEW

File: src/avrdude.conf.in [hunk 1]
```diff
@@ -2091,6 +2153,13 @@ programmer # butterfly_mk
 # jtagmkI
 #------------------------------------------------------------
 
+# The original AVR JTAG ICE. It only support a handful of
```
Comment (stefanrueger) [inline src/avrdude.conf.in:2156]: supports

**Comment by stefanrueger** [comment]
Thanks! This is  a brilliant and extensive improvement of the conf file. Only found one typo. Will merge before v7.3 reease.

**Comment by mcuee** [comment]
Yes, I agree this is a great improvement!

**Comment by MCUdude** [comment]
@stefanrueger @mcuee I've added information about older parallel port-based programmers as well. Now most, if not all programmers have decent information in the avrdude.conf file.

Is it enough to close #1437?

**Comment by mcuee** [comment]
@MCUdude 

Yes I think it is okay to use this PR to close #1437.

* #1437

**Pull request merged**